package.json

```diff
@@ -1,29 +1,98 @@
+          "pressed": "config.sonarqube.showIssuesOnCodeViews"
+        }
+      }
+    ],
     "menus": {
-      "editor/title": [],
+      "editor/title": [
+        {
+          "action": "sonarqube.showIssuesOnCodeViews.toggle",
+          "when": "resource"
+        }
+      ],
       "commandPalette": []
     },
-    "configuration": {}
+    "configuration": {
+      "type": "object",
+      "properties": {
+        "sonarqube.showIssuesOnCodeViews": {
+          "description": "Whether to show Sonarqube issues inline on code views.",
+          "type": "boolean",
+          "default": true
+        },
+        "sonarqube.instanceUrl": {
+          "description": "The URL to the Sonarqube instance.",
+          "type": "string",
+          "default": "https://sonarcloud.io/"
+        },
+        "sonarqube.corsAnywhereUrl": {
+          "description": "The URL to a CORS proxy.",
+          "type": "string",
+          "default": "https://cors-anywhere.herokuapp.com"
+        },
+        "sonarqube.repositoryNamePattern": {
+          "description": "Regular expression with that is matched on the repository name to extract the capture groups for organization and project key templates.",
+          "type": "string",
+          "format": "regex",
+          "default": "(?:^|/)([^/]+)/([^/]+)$"
+        },
+        "sonarqube.organizationKeyTemplate": {
+          "description": "Replace string to build the organization key from the repository name pattern, using $n references for capture groups. By default just uses the first capture group",
+          "type": "string",
+          "default": "$1"
+        },
+        "sonarqube.projectKeyTemplate": {
+          "description": "Replace string to build the project key from the repository name pattern, using $n references for capture groups.",
+          "type": "string",
+          "default": "$1_$2"
+        }
+      }
+    }
   },
   "version": "0.0.0-DEVELOPMENT",
   "license": "Apache-2.0",
   "main": "dist/sonarqube.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/sourcegraph/sourcegraph-sonarqube"
+  },
   "scripts": {
     "eslint": "eslint 'src/**/*.ts'",
     "typecheck": "tsc -p tsconfig.json",
-    "build": "parcel build --out-file dist/sonarqube.js src/sonarqube.ts",
+    "build": "parcel build --out-file dist/sonarqube.js src/sonarqube.ts --no-minify",
```

Comment:
Why no minify?

Reply:
I faced an error when building because terser (the minifier used by Parcel) does not support nullish coalescing yet. I tried configuring Babel to always transform it, but faced another Parcel bug that discarded my Babel config.